htdocs/accountancy/bookkeeping/list.php

```diff
@@ -360,6 +377,22 @@
 		$accountancyexport = new AccountancyExport($db);
 		$accountancyexport->export($object->lines);
 
+        // TODO Move in class bookKeeping
+        // Specify as export : update field date_export
+        foreach ( $object->lines as $movement ) {
+            $now = dol_now();
+            $sql = " UPDATE " . MAIN_DB_PREFIX . "accounting_bookkeeping";
+            $sql .= " SET date_export = '" . $db->idate($now) . "'";
+            $sql .= " WHERE rowid = " . $movement->id;
+
+            dol_syslog("/accountancy/bookeeping/list.php Function export_file Specify movements as exported sql=" . $sql, LOG_DEBUG);
+            if ($db->query($sql)) {
+                $db->commit();
```

Comment:
If you add some code with a commit or rollback, you must also add code (on same level of indentation) that make a begin()

Reply:
Code update